packages/components/src/components/checklist/checklist.tsx

```diff
@@ -28,13 +34,21 @@ const ItemStatus = ({ status, onClick, button_text }) => {
     }
 };
 
-ItemStatus.propTypes = {
-    button_text: PropTypes.string,
-    onClick: PropTypes.func,
-    status: PropTypes.string,
+type TChecklistItem = {
+    is_disabled: boolean;
+    content: string;
+    status: string;
+    button_text: string;
+    onClick: () => void;
 };
```

Comment:
```suggestion
type TChecklistItem = TItemStatusProps & {
    is_disabled: boolean;
    content: string;
};
```
@iman-fs it's preferable to move all types declarations to the top of the files. would you please move it? 🙂

Reply:
you're right. I'll added the changes. thank you